[E]69.cpp

```diff
@@ -0,0 +1,19 @@
+// Problem: 69. Sqrt(x) - Easy
+// Link: https://leetcode.com/problems/sqrtx/
+
+class Solution {
+public:
+    int mySqrt(int x) {
+        // cout << (int)sqrt(INT_MAX) << endl;
+        // I must cout to find the max of range first, before searching.
+        int l = 0, r = 46340;
+        while (l <= r) {
+            int mid = (l + r) / 2;
```

Comment:
Đẹp nhất vẫn là mid = l + (r - l) / 2; Để tránh bị tràn số khi l và r quá to.

Reply:
Thank you. Brilliant idea!